apps/frontend/src/app/containers/project/InsightProjectNew.tsx

```diff
@@ -13,7 +13,7 @@ const defaultProject: IInsightProject = {
     description: '',
     start: '',
     end: '',
-    members: [],
+    members: []
 };
 
 export const InsightProjectNew = (): ReactElement => {
```

Comment:
Maby rename to CreateInsightProject, if that is what the component is intended for